index.html

```diff
@@ -8499,6 +8499,16 @@ <h3>Value</h3>
 		<h2>Values for States and Properties</h2>
 		<p>Many <a>states</a> and <a>properties</a> accept a specific set of tokens as <span>values</span>. The allowed values and explanation of their meaning is shown after the table of characteristics. The default value, if defined, is shown in <strong>strong</strong> type, followed by the parenthetical term 'default'. When a value is indicated as the default, the user agent MUST follow the behavior prescribed by this value when the state or property is empty or unspecified. Some <a>roles</a> also define what behavior to use when certain states or properties, that do not have default values, are not provided.</p>
 	</section>
+	<section>
+		<h2>Translatable States and Properties</h2>
+		<p>The values of the following <a>states</a> and <a>properties</a> should be translated when a page is localized:</p>
```

Comment:
Perhaps:

>In order to be understandable by assistive technology users, the values of the following states and properties need to be translated when a page is localized:

This revision:
1. clarifies the reason for translation.
2. Avoids normative sounding language when there is no clear subject for a normative requirement.